images/win/scripts/Installers/Install-Docker.ps1

```diff
@@ -5,24 +5,26 @@
 ##         can continue.
 ################################################################################
 
-# Docker EE 20.10.8 has the regression
-# fatal: open C:\ProgramData\docker\panic.log: Access is denied.
-Write-Host "Install-Package Docker"
-Install-Package -Name docker -ProviderName DockerMsftProvider -RequiredVersion 20.10.7 -Force
-Start-Service docker
+Write-Host "Install Docker CE"
+$instScriptUrl = "https://raw.githubusercontent.com/microsoft/Windows-Containers/Main/helpful_tools/Install-DockerCE/install-docker-ce.ps1"
+$instScriptPath = "$env:TEMP\install-docker-ce.ps1"
+Invoke-WebRequest -UseBasicParsing $instScriptUrl -o $instScriptPath
```

Comment:
nitpick: replace to Start-DownloadWithRetry or use full param names(Output,Uri,etc)

Reply:
Makes sense, replaced, thank you!